.github/dependabot.yml

```
@@ -1,97 +1,9 @@
 version: 2
 updates:
   - package-ecosystem: "npm"
-    directory: "/fixtures/art"
+    directory: "/fixtures/*"
```

Needs to be directories
```suggestion
    directories: "/fixtures/*"
```